qutebrowser/completion/completer.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, cmd, win_id, parent=None):
         self._win_id = win_id
         self._cmd = cmd
         self._cmd.update_completion.connect(self.schedule_completion_update)
+        self._cmd.update_completion_now.connect(self.update_completion)
```

Comment:
Note to self: I probably should update those signals after this is merged:

* `update_completion` -> `schedule_completion_update`
* `update_completion_now` -> `update_completion`

I guess that would've reduced your confusion?